docs/try/run-a-node.md

```diff
@@ -54,9 +52,9 @@ supported CLI commands][CLI].
 ## Open your ports!
 
 The best way to contribute to the growth and sustainability of the
-Witnet network is **opening up the listening port of your node** to the
-open Internet, so that other nodes in the network can download block
-chain data from yours and **your transactions can be broadcast more
+Witnet network is by **opening up the listening port of your node**, 
+so that other nodes in the network can download block
+chain data from you and **your transactions can be broadcasted more
```

Comment:
https://www.merriam-webster.com/words-at-play/is-it-broadcast-or-broadcasted-forecast-or-forecasted-usage

Reply:
I've always puzzled by the lack of clarity on how to build the past participle of broadcast.

Reply:
I would _use_ "broadcasted" in conversation, which is what the article you posted suggests. But yes, another weird English anomaly.